project-codeflare/codeflare-operator
# Add Ray cluster REST API support in test support package

# Issue link
Part of https://github.com/opendatahub-io/distributed-workloads/issues/63

# What changes have been made
Adjustments in e2e test support package to provide support for new ODH integration test - https://github.com/opendatahub-io/distributed-workloads/pull/71

# Verification steps
The changes can be verified by running ODH integration test, see instructions in https://github.com/opendatahub-io/distributed-workloads/pull/71

## Checks
- Testing Strategy

It is enough to run the local e2e tests (part of the PR check) to make sure nothing is broken.
